opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx

```diff
@@ -233,7 +233,7 @@ const CaseIncidentDetails: FunctionComponent<CaseIncidentDetailsProps> = ({
           {t('Correlated cases')}
         </Typography>
         <List classes={{ root: classes.relatedContainers }}>
-          {expandable
+          {(data.relatedContainers?.edges ?? []).length >= 2
```

Comment:
I think you can write something like this instead :
`data.relatedContainers?.edges?.length >= 2`